lib/ThreeScaleClient.php

```diff
@@ -257,10 +306,21 @@ public function authorize_with_user_key($userKey, $serviceId = null, $usage = nu
    * </code>
    */
 
-   public function authrep($appId, $appKey = null, $usage = null, $userId = null, $object = null, $no_body = null, $serviceId = null) {  
+   public function authrep($appId, $appKey = null, $serviceId = null, $usage = null, $userId = null, $object = null, $no_body = null) {  
```

Comment:
This is a breaking change no? Moving all parameters by one. Whoever updates will get invalid calls unless they add serviceId too ?